skbeam/core/accumulators/binned_statistic.py

```diff
@@ -72,6 +72,9 @@ def __init__(self, sample, statistic='mean',
             minimum and maximum values along each dimension.
 
         """
+        if mask is None:
+            mask = np.ones_like(sample)
```

Comment:
This is already documented in the threshold mask you wrote, [here](https://github.com/scikit-beam/scikit-beam/blob/master/skbeam/core/mask.py#L83)